### PR TITLE
Document Code Owner Responsibilities and Repo Maintainer Procedures 

### PR DESCRIPTION
**Pull request by MichaelSimons**
Fixes https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/972

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,11 +243,36 @@ To support this scenario, the manifest entry for the base image must be defined
...
+### Code Owner Responsibilities
+
+- **Code reviews** - Code review all changes made to the owned Dockerfiles.
+- **Respond to build breaks** - It is possible for a Dockerfile to regress and cause a build break.
+When this occurs, the code owner is responsible for addressing the break.
+See [Responding to Build Breaks](#responding-to-build-breaks) for additional details.
+- **Address CVEs** - When fixable CVEs are reported that require Dockerfiles changes, the code owner is responsible for mitigating the CVE.
+- **Remove EOL distros** - Remove or update Dockerfiles based on EOL distros.
+
+Wheneven a Dockerfile change is required:
```
Comment (mthalman) [inline README.md:255]:
```suggestion
Whenever a Dockerfile change is required:
```

File: README.md [hunk 2]
```diff
@@ -242,11 +243,36 @@ To support this scenario, the manifest entry for the base image must be defined
...
 
+## Repo Maintainer Procedures
+
+### Responding to Build Breaks
+
+When a Dockerfile build break occurs the following steps are to be taken:
+
+1. Create a new issue detailing the break.  @mention the appropriate [CODEOWNER](./CODEOWNERS).
+2. If the break is blocking automatic rebuilds or blocking the development/release of new changes (judgement decision),
+then disable the Dockerfile from building by commenting out the [manifest entry](#manifest).
```
Comment (mthalman) [inline README.md:273]: I don't know if commenting out works (with the question of whether comments are supported in JSON by the reader). AFAIK, we haven't tested that scenario. Deletion should probably be the recommendation for now.

Comment (MichaelSimons) [reply README.md]: I adjusted the step to delete the manifest entry in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1194/commits/7c327dc2cfe91fc1c1370d08fa4fe53852e43dc2.

File: README.md [hunk 3]
```diff
@@ -242,11 +243,36 @@ To support this scenario, the manifest entry for the base image must be defined
...
 
+## Repo Maintainer Procedures
+
+### Responding to Build Breaks
+
+When a Dockerfile build break occurs the following steps are to be taken:
+
+1. Create a new issue detailing the break.  @mention the appropriate [CODEOWNER](./CODEOWNERS).
+2. If the break is blocking automatic rebuilds or blocking the development/release of new changes (judgement decision),
+then disable the Dockerfile from building by deleteing the [manifest entry](#manifest).
```
Comment (mthalman) [inline README.md:273]:
```suggestion
then disable the Dockerfile from building by deleting the [manifest entry](#manifest).
```

**Review by mthalman — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (MichaelSimons) [reply README.md]: I missed part of my changes (unsaved).

File: README.md (hunk 1 above)
Comment (mthalman) [reply README.md:255]: Now you reverted this change. 😄

Comment (MichaelSimons) [reply README.md]: Goodness!

Comment (MichaelSimons) [reply README.md]: Added back.

**Review by mthalman — APPROVED** [review]

**Pull request merged**